packages/alfa-rules/test/sia-r8/rule.spec.tsx

```diff
@@ -0,0 +1,200 @@
+import { test } from "@siteimprove/alfa-test";
+
+import { Document } from "@siteimprove/alfa-dom";
+
+import R8, { Outcomes } from "../../src/sia-r8/rule";
+
+import { evaluate } from "../common/evaluate";
+import { passed, failed, inapplicable } from "../common/outcome";
+
+test("evaluate() passes an input element with accessible name", async (t) => {
```

Comment:
Let's be more precise on how it gets its name ("implicit" because it is a surrounding element).
```suggestion
test("evaluate() passes an input element with implicit label", async (t) => {
```